src/api/mod.rs

```diff
@@ -1,13 +1,12 @@
 //! `Web3` implementation
 
 mod eth;
-mod eth_filter;
+pub mod eth_filter;
```

Comment:
That's bad, better re-export everything for convenience. We don't really need to expose the module structure details imho.